docs/docs/cmd/teams/meeting/meeting-add.mdx

```diff
@@ -0,0 +1,277 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# teams meeting add
+
+Create a new online meeting
+
+## Usage
+
+```sh
+m365 teams meeting add [options]
+```
+
+## Options
+
+```md definition-list
+`-s, --startTime [startTime]`
+: The start time of the meeting. If not specified, the startTime will be set to the current time.
+
+`-e, --endTime [endTime]`
+: The end time of the meeting. If not specified, the endTime will be set to one hour after the startTime.
+
+`--subject [subject]`
+: The subject of the meeting.
+
+`-p, --participantUserNames [participantUserNames]`
+: A comma-separated list of participant UPNs.
+
+`--organizerEmail [organizerEmail]`
+: The organizer's email address.
+
+`-r, --recordAutomatically`
+: When using this flag, the meeting will be recorded automatically.
+```
+
+<Global />
+
+## Remarks
+
```

Comment:
For everybody who will be confused: maybe we should include the note of the officials graph docs here as well:

> This API creates a standalone meeting that is not associated with any event on the user's calendar; therefore, meetings created via this API will not show on the user's calendar.

Reply:
Okay, you are right; it could be a surprise for a user that creation gives no visual effect :)

Reply:
Nice, I just copy pasted the graph note by the way. I now see it should read "This command"

Also, in this case I think we should show the note in a ::: info ::: section